packages/web-components/fast-foundation/src/disclosure/disclosure.spec.ts

```diff
@@ -17,21 +18,15 @@ async function createDisclosure() {
     return { element, connect, disconnect };
 }
 
-async function macrotask() {
-    return new Promise((resolve, reject) => {
-        window.setTimeout(() => {
-            resolve(void 0);
-        })
-    })
-}
+
 
 describe("Disclosure", () => {
     describe("User interaction", () => {
         it("should toggle the content using `toggle()`", async () => {
             const { element, connect, disconnect } = await createDisclosure();
             await connect();
             element.toggle();
-            await macrotask();
+            await timeout();
```

Comment:
So does having a timeout of *0* effectively just create an empty task/promise to simulate some work being done?

Reply:
A timeout of 0 will queue a JavaScript "task" (sometimes called a macro task) with no delay. This pattern provides all the queued micro tasks (like template changes) an opportunity to execute prior to moving onto the next statement. https://developer.mozilla.org/en-US/docs/Web/API/HTML_DOM_API/Microtask_guide#tasks_vs_microtasks is a good read to understand what is going on.